projects/go-lib/src/lib/components/go-icon/go-icon.component.scss

```diff
@@ -32,21 +32,21 @@ $icon-sizes: (small: 1rem, medium: 1.5rem, large: 2rem);
 }
 
 .go-icon--dark {
-  color: $theme-dark-color;
+  color: $theme-light-color;
```

Comment:
@jaredami I think instead of changing this. We can just remove the `[iconModifier]` being passed to the `go-icon` by the `go-button`. See line 18 in `go-button.component.html`. The inheritance should take over here.